pkg/ansibleapp/types_test.go

```diff
@@ -16,17 +16,20 @@ const SpecDescription = "A note taking webapp"
 
 var SpecJSON = fmt.Sprintf(`
 {
-  "id": "%s",
+	"id": "%s",
 	"description": "%s",
-  "name": "%s",
-  "bindable": %t,
-  "async": "%s"
+	"name": "%s",
+	"bindable": %t,
+	"async": "%s"
```

Comment:
Editor was complaining about mismatched indentation. It saw 2 spaces here in a file that is tabbed (since I copied this in from a json file where I had 2 space tab settings). Updated here for uniformity.